source/guide/3.2/webui-blueprints-page.md

```diff
@@ -13,15 +13,31 @@ terminology_link: reference-terminology.html
 When clicking on the `Blueprints` tab and choosing a blueprint you will be able to choose one of the following:
 
 # Topology
-See the definition [here]({{page.terminology_link}}#topology).<br/>
```

Comment:
Most of the views in both blueprints & deployments are pretty much the same, but there are a few small differences. In deployment views we show the runtime properties instead of the static properties in the blueprint views.